spec.emu

```diff
@@ -331,6 +359,21 @@ contributors: Google, Ecma International
         The value of the [[RelevantExtensionKeys]] internal slot is &laquo;  &raquo;.
       </p>
 
+      <p>
+        The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints:
+      </p>
+
+      <ul>
+        <li>[[LocaleData]].[[&lt;_locale_&gt;]] must have a types field for all locale values _locale_.  The value of this field must be a Record, which must have fields with the names of the six display name types: `"language"`, `"region"`, `"script"`, `"currency"`, `"dateTime"`, and `"dateTimeField"`.</li>
+        <li>The value of each of those fields in turn must be a Records which must have fields with the names of the three display name styles: `"narrow"`, `"short"`, and `"long"`.</li>
```

Comment:
same here, and throughout

Reply:
could you point me to an example in either ECMA262 or ECMA402 that I can follow?